Tests/PackageModelTests/SwiftSDKTests.swift

```diff
@@ -348,15 +348,15 @@ final class DestinationTests: XCTestCase {
             otherToolsNoRoot,
             someToolsWithRoot,
             invalidToolset,
-        ] {
-            try fs.writeFileContents(AbsolutePath(validating: testFile.path), string: testFile.json)
+        ] as [(path: AbsolutePath, json: SerializedJSON)] {
```

Comment:
Why not move this into the variable declaration?

```
for testFile: [(AbsolutePath, SerializedJSON)] in [
  ...
] {
  try fs.writeFileContents($0.path, string: $0.json.underlying)
}

Reply:
Separated out into a file-local `private let` declaration in the latest commit.